docs/basics/firststeps.md

```diff
@@ -20,11 +20,15 @@ To delete the current annotation in progress, click the **Clear Annotation** but
 To select a different annotation click the **Select** button, click an area of the desired annotation and select the desired functionality to alter.
 To undo the last action, click the **Undo last action** and to redo the action, click the **Redo last action**.
 
-Once annotations have been completed, click the **Save Annotations** button to confirm your changes.
-Please note: if you leave annotate without saving then your annotations will not be added to the image metadata.
+At any point, click the **Save Annotations** button to save your progress.
```

Comment:
Maybe in the image above we could add some tooltips/text with arrows saying which button is which?